doc/classes/TextEdit.xml

```diff
@@ -656,7 +656,7 @@
 		<method name="get_total_visible_line_count" qualifiers="const">
 			<return type="int" />
 			<description>
-				Returns the number of lines that may be drawn.
+				Returns the number of lines in the text, including wrapped lines and excluding folded lines. If [member wrap_mode] is set to [constant LINE_WRAPPING_NONE] and no lines are folded (see [method CodeEdit.is_line_folded]) then this is equivalent to [method get_line_count]. See [method get_visible_line_count_in_range] for a limited range of lines.
```

Comment:
I want to avoid `may be drawn` since the lines it counts may not be visible (ex. lines that to be scrolled down to).
Also it sort of implies to be the number of lines that fit in the control, but there can be less than that.

Reply:
I see. My main issue is that there's no summary of what this method considers. The function signature is not localized, so it can only be relied upon so much.
But it's not that bad, saying "total" may be just fine:
```suggestion
				Returns the total number of lines in the text. This includes wrapped lines and excludes folded lines. See [method get_visible_line_count_in_range] for a limited range of lines.
				[b]Note:[/b] If [member wrap_mode] is set to [constant LINE_WRAPPING_NONE] and no lines are folded (see [method CodeEdit.is_line_folded]) then this is equivalent to [method get_line_count].
```